ui/components/multichain/multichain-token-list/multichain-token-list.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import classnames from 'classnames';
+import {
+  DISPLAY,
+  FLEX_DIRECTION,
+  FONT_WEIGHT,
+  JustifyContent,
+  Size,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarNetwork,
+  AvatarToken,
+  AvatarWithBadge,
+  AVATAR_WITH_BADGE_POSTIONS,
+  Text,
+} from '../../component-library';
+import Box from '../../ui/box/box';
+import { getNativeCurrencyImage } from '../../../selectors';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const MultichainTokenList = ({
+  className,
+  onClick,
+  tokenSymbol,
+  tokenImage,
+  primary,
+  secondary,
+}) => {
+  const primaryTokenImage = useSelector(getNativeCurrencyImage);
+  const t = useI18nContext();
+  return (
+    <Box
+      className={classnames('multichain-token-list', className)}
+      display={DISPLAY.FLEX}
+      flexDirection={FLEX_DIRECTION.COLUMN}
+      gap={4}
+    >
+      <Box
+        className="multichain-token-list__container-cell"
+        display={DISPLAY.FLEX}
+        flexDirection={FLEX_DIRECTION.ROW}
+        padding={4}
+        as="a"
+        href="#"
+        onClick={onClick}
+      >
+        <AvatarWithBadge
+          badgePosition={AVATAR_WITH_BADGE_POSTIONS.TOP}
+          badge={
+            <AvatarNetwork
+              size={Size.XS}
+              name={t('networkNameEthereum')}
+              src={primaryTokenImage}
+            />
+          }
+          marginRight={3}
+        >
+          <AvatarToken src={tokenImage} showHalo />
```

Comment:
Missing name prop here
```suggestion
          <AvatarToken name={tokenSymbol}  src={tokenImage} showHalo />
```